<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reception</class>
 <widget class="QWidget" name="Reception">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonCheckIn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开房</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCheckOut">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退房</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCheckOut">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextInfo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCheckIn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>UserID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>RoomID:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
